client/src/pages/Register/PageWrapper.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
```

Comment:
Although this wrapper for the register pages was not exactly necessary, this prevents me from having to refactor all the stylesheets to some new hierarchy.